### ISSUE TITLE
Client and admin container exits with code 0

### ISSUE THREAD

**Issue by n0wy** [api-platform/api-platform#1503]
**API Platform version(s) affected**: 2.5.4

**Description**  
client and admin container exits with code 0 after `docker-composer up`

`docker-compose ps` gives me:
```
         Name                       Command                  State                   Ports            
------------------------------------------------------------------------------------------------------
api-platform_admin_1     docker-entrypoint.sh yarn  ...   Exit 0                                      
api-platform_api_1       /usr/local/openresty/bin/o ...   Up                                          
api-platform_client_1    docker-entrypoint.sh yarn  ...   Exit 0                                      
api-platform_db_1        docker-entrypoint.sh postgres    Up             0.0.0.0:5432->5432/tcp       
api-platform_dev-tls_1   nginx -g daemon off;             Up             0.0.0.0:80->80/tcp           
api-platform_mercure_1   /mercure                         Up             0.0.0.0:1337->443/tcp, 80/tcp
api-platform_php_1       docker-entrypoint php-fpm        Up (healthy)   9000/tcp                     
api-platform_vulcain_1   ./vulcain                        Up             0.0.0.0:8443->443/tcp, 80/tcp
```

Running composer-up for admin and client:
```
n0wy:~/code/api-platform (tags/v2.5.4^0 ?) $ docker-compose up client
api-platform_dev-tls_1 is up-to-date
Starting api-platform_client_1 ... done
Attaching to api-platform_client_1
client_1   | yarn run v1.22.4
client_1   | $ react-scripts start
client_1   | ℹ ｢wds｣: Project is running at https://172.21.0.8/
client_1   | ℹ ｢wds｣: webpack output is served from 
client_1   | ℹ ｢wds｣: Content not from webpack is served from /usr/src/client/public
client_1   | ℹ ｢wds｣: 404s will fallback to /
client_1   | Starting the development server...
client_1   | 
client_1   | Done in 1.28s.
api-platform_client_1 exited with code 0
n0wy:~/code/api-platform (tags/v2.5.4^0 ?) $ docker-compose up admin
api-platform_dev-tls_1 is up-to-date
Starting api-platform_admin_1 ... done
Attaching to api-platform_admin_1
admin_1    | yarn run v1.22.4
admin_1    | $ react-scripts start
admin_1    | ℹ ｢wds｣: Project is running at https://172.21.0.8/
admin_1    | ℹ ｢wds｣: webpack output is served from 
admin_1    | ℹ ｢wds｣: Content not from webpack is served from /usr/src/admin/public
admin_1    | ℹ ｢wds｣: 404s will fallback to /
admin_1    | Starting the development server...
admin_1    | 
admin_1    | Done in 1.31s.
api-platform_admin_1 exited with code 0
```

**Additional Context**  

Docker version and os
```
n0wy:~/code/api-platform (tags/v2.5.4^0 ?) $ docker-compose -v
docker-compose version 1.25.5, build 8a1c60f6
n0wy:~/code/api-platform (tags/v2.5.4^0 ?) $ docker -v
Docker version 19.03.4-ce, build 9013bf583a
n0wy:~/code/api-platform (tags/v2.5.4^0 ?) $ lsb_release -a
LSB Version:	1.4
Distributor ID:	Arch
Description:	Arch Linux
Release:	rolling
Codename:	n/a
```

**Comment by bertrandjamin** [comment]
@n0wy 

I have encountered the same problem after a first installation. 

Did you launched the command : `docker-compose pull` before `docker-compose up -d` (https://api-platform.com/docs/distribution/#installing-the-framework) ? 
I didn't do that personnally so I expect that the problem come from this...

On my side I have resolved those errors by forcing the build of the containers admin & client 

`docker-compose up --build -d admin && docker-compose up --build -d client`

Now I can't go back to the previous bugged stage (despite a totally reseted docker env.)  so I can't validate my first advice.

But at least, now, I can continue the "Getting Started" \o/

I hope that this answer can help you...

**Comment by prodigeris** [comment]
Same here.

I've done `docker-compose pull` before starting containers; that did not help.

Rebuilding containers helped though!

**Comment by n0wy** [comment]
@bertrandjamin 
I did followed the docks and from what I remember there was no error on any of those commands.
Tried `ocker-compose up --forece-recreate` and then `docker-compose pull` and `docker-compose up -d` but issue persisted.

I did tried your advice and run `docker-compose up --build -d admin && docker-compose up --build -d client` today and it fixed it! I've noticed it took quite some time comparing to initial run. I think it did something extra which didn't happen then.

Thanks! You saved me many hours of debugging
